kurtosis.yml

```diff
@@ -1 +1 @@
-name: "github.com/ethpandaops/ethereum-package"
+name: "github.com/rebelArtists/ethereum-package"
```

Comment:
looks good , but pls change this back

Reply:
yep, was draft for testing, reverted back to original